scripts/get_latest_dockerhub_version.py

```diff
@@ -22,4 +26,12 @@
         numeric_tags.append(tag_to_store)
 
 numeric_tags.sort(key=lambda s: list(map(int, s.split('.'))), reverse=True)
-print(numeric_tags[0])
+
+if check_version == "latest":
+    print(numeric_tags[0]) # print latest version
+else:
```

Comment:
A little confusing at first how this function now returns either tag name of latest version or true false, based on whether the "latest" check_version argument is used. But I like it. It makes sense after thinking about it.